uaclient/cli.py

```diff
@@ -1290,8 +1291,10 @@ def action_auto_attach(args, *, cfg):
 
     try:
         actions.auto_attach(cfg, instance)
-    except exceptions.UrlError:
-        event.info(messages.ATTACH_FAILURE)
+    except (exceptions.UrlError, HTTPError) as e:
+        if isinstance(e, HTTPError):
+            event.info(getattr(e, "msg", None))
+        event.info(messages.ATTACH_FAILURE.msg)
```

Comment:
Right now, the error when running `auto-attach` is:
```
Service account not enabled on this instance
Failed to attach machine. See https://ubuntu.com/advantage
```

I think we don't need the second part of the message here, since going https://ubuntu.com/advantage will not help at all.

I propose we change it to something like that:
```
Failed to attach machine
Required service account information missing in the instance.
For more information, see: https://cloud.google.com/iam/docs/service-accounts
```

Now, we do have the problem that the google url can change in the future and we would need an SRU for that.
What do you think @orndorffgrant ?